modules/stanford_basic_page_types/templates/components/basic_page_types_list/basic-page-types-list.twig

```diff
@@ -0,0 +1,52 @@
+{#
+/**
+ * @file
+ * Basic Page Types List
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - basic_page_types_category: The taxonomy term for the type.
+ * - basic_page_types_title: The Page title.
+ * - basic_page_types_description: The page description.
+ * - basic_page_types_page_image: the page image.
+ * - basic_page_types_footer: an open footer.
+ */
+#}
+{%- set attributes = attributes.addClass(['su-basic-page-types-list', modifier_class]) -%}
+
+<ul{{ attributes }}>
```

Comment:
`ul` and `li` elements arent appropriate here. these should all be divs

Reply:
I took that structure and the associated styles directly from template for the news list.  I thought it was strange too, but I figured following the established pattern was better than re-working it.  If it's not right here, it's probably not right in the news list either.

Reply:
This is fixed. Changed to divs.  I didn't fix news though.